docs/dev.md

```diff
@@ -0,0 +1,23 @@
+Dev
+===
+
```

Comment:
Can you add commands and information about:

 - where the executables are coming from (e.g., to wget the jar)
 - where this is being run from in context of the repository
 - on a cluster? Local machine? I would want to be able to copy paste and do this on my own relatively easily.

Reply:
will update it

Reply:
werd!